sa/authz.go

```diff
@@ -0,0 +1,82 @@
+package sa
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	gorp "github.com/letsencrypt/boulder/Godeps/_workspace/src/gopkg.in/gorp.v1"
+
+	"github.com/letsencrypt/boulder/core"
+)
+
+var authorizationTables = []string{
+	"authz",
+	"pendingAuthorizations",
+}
+
+const getAuthorizationIDsMax = 1000
+
+func getAuthorizationIDsByDomain(db *gorp.DbMap, ident string, now time.Time) ([]string, error) {
```

Comment:
I might just add a table string to this and call it twice in Revoke. Could even do it in a goroutine for the two different tables and do both at the same time.